backend/src/routes/v0/thread.py

```
@@ -223,6 +224,47 @@ async def get_thread(
+    """
+    try:
+        return StreamingResponse(
+            stream_from_redis(thread_id),
+            media_type="text/event-stream",
+            headers={
+                "Cache-Control": "no-cache",
+                "Connection": "keep-alive",
+                "X-Accel-Buffering": "no",  # Disable nginx buffering
+            },
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)